pkg/remote/remote_test.go

```diff
@@ -817,3 +835,25 @@ func TestCreateDockerignoreFileWithFilesystem(t *testing.T) {
 		})
 	}
 }
+
+func TestGetOktetoPrefixEnvVars(t *testing.T) {
```

Comment:
nit: i've tested with lowecase env... and its not selected. We enforce uppercase only or shall we consider `okteto_env` also a possible env? 🤔

Reply:
good point, I think it could be an edge case that if someone cannot work around we might consider adding

Reply:
I don't think it's worth adding them. This feature was for the feature flag where we require them to be uppercase